pkg/wallet/wallet.go

```diff
@@ -307,16 +310,26 @@ func LoadFile(filePath string) (Wallet, *WalletError) {
 	if err != nil {
 		return Wallet{}, &WalletError{fmt.Errorf("unmarshalling file '%s': %w", filePath, err), utils.ErrAccountFile}
 	}
-
+	walletHasNickname := len(accountSerialized.Nickname) > 0
+	_, nicknameFromFileName := filepath.Split(filePath)
+	nicknameFromFileName = strings.TrimPrefix(nicknameFromFileName, "wallet_")
+	nicknameFromFileName = strings.TrimSuffix(nicknameFromFileName, ".yaml")
```

Comment:
you can extract this into a function near https://github.com/massalabs/station-massa-wallet/blob/86040bea9ca6414c2087770de7968e2b08548028/pkg/wallet/wallet.go#L389

Reply:
```go
```

Reply:
you can create this function and put it bellow that function: https://github.com/massalabs/station-massa-wallet/blob/86040bea9ca6414c2087770de7968e2b08548028/pkg/wallet/wallet.go#L389

```go
func NicknameFromFilePath(filePath string): string {
	_, nicknameFromFileName := filepath.Split(filePath)
	nicknameFromFileName = strings.TrimPrefix(nicknameFromFileName, "wallet_")
	nicknameFromFileName = strings.TrimSuffix(nicknameFromFileName, ".yaml")

	return nicknameFromFileName
}
```

Reply:
in golang, we usually don't prefix the getter functions with `get`